xonsh/xoreutils/__init__.py

```diff
@@ -0,0 +1,6 @@
+from xonsh.xoreutils.echo import echo
+from xonsh.xoreutils.pwd import pwd
+from xonsh.xoreutils.cat import cat
+from xonsh.xoreutils.tty import tty
+from xonsh.xoreutils.tee import tee
+from xonsh.xoreutils.yes import yes
```

Comment:
From Pep8

Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path ):

import mypkg.sibling
from mypkg import sibling
from mypkg.sibling import example